documentation/docs/04-loading.md

```diff
@@ -144,6 +144,8 @@ This will be merged with any existing `stuff` and passed to the `load` functions
 
 The combined `stuff` is available to components using the [page store](/docs/modules#$app-stores) as `$page.stuff`, providing a mechanism for pages to pass data 'upward' to layouts.
 
+> `status`, `error`, `redirect` and `cache` are ignored when rendering error pages.
```

Comment:
putting this note in `stuff` seems like a surprising place for it. maybe under `status` would work?

Reply:
oh, this was supposed to go at the end of `Output`, but i managed to put it before `dependencies`. i think it probably belongs at the end or at the top, rather than between items

Reply:
end of `Output` sounds good too